lib/rucio/core/replica.py

```diff
@@ -739,27 +739,47 @@ def _list_replicas(dataset_clause, file_clause, state_clause, show_pfns, schemes
                     rse_schemes = schemes or []
                     if not rse_schemes:
                         try:
-                            rse_schemes = [rsemgr.select_protocol(rse_settings=rse_info[rse],
-                                                                  operation='read',
-                                                                  domain=domain)['scheme']]
+                            if domain == 'all':
+                                rse_schemes.append(rsemgr.select_protocol(rse_settings=rse_info[rse],
+                                                                          operation='read',
+                                                                          domain='wan')['scheme'])
+                                rse_schemes.append(rsemgr.select_protocol(rse_settings=rse_info[rse],
+                                                                          operation='read',
+                                                                          domain='lan')['scheme'])
```

Comment:
the same protocol can be defined on wan and lan which will can create some duplicates in the rse_schemes list.